db.js

```diff
@@ -1,10 +1,15 @@
 const Pool = require ('pg').Pool;
+require('dotenv').config();
+
+const { USER, HOST, DATABASE, PASSWORD } = process.env;
+
+
 const pool = new Pool ({
-  user: "q_and_a_service",
-  host: "localhost",
-  database: "q_and_a_service",
-  password: "#sdc#",
-  port: 5432
+  user: USER,
```

Comment:
Just a note. When using `USER` here I ran into a conflict where the value for `USER` defined in my `.env` file was getting overwritten by the local system's value for `USER`.

To get around this, I changed the variable to something like `DB_USER` which avoided the conflicting variables and prevented the value from being over written.

Not sure if this is something you will run into, but it is something to watch out for

Reply:
Thanks for this!  I will make the change just to be on the safe side.